plugins/alert-handlers/snmp-alerts/src/org/apache/cloudstack/alert/snmp/SnmpHelper.java

```diff
@@ -65,6 +66,7 @@ private PDU createPDU(SnmpTrapInfo snmpTrapInfo) {
 
         int alertType = snmpTrapInfo.getAlertType() + 1;
         if (alertType > 0) {
+            trap.add(new VariableBinding(SnmpConstants.sysUpTime, new OctetString(new Date().toString())));
```

Comment:
Agree with @DaanHoogland here. sysuptime isn't indeed now(), but for how long the system has been up.